### PR TITLE
Adding bom

### PR DESCRIPTION
**Pull request by etsauer**
cc @redhat-cop/containerize-it

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  ** Terminal commands
+  ** etc.
+* Presenter Notes
+  ** Guidelines for "what to {type|click|do}" to walk through the demo
+  ** Guidelines for "what to say" while giving the demo (level of detail will vary depending on the audience and intended presenter)
+  ** Interesting concepts, anecdotes to point out along the way
+
+== Quickstart Model Example
+
+For a good example of what a quickstart should look like, see the `s2i-play` directory as an example.
```
Comment (sabre1041) [inline README.adoc]: Can you make `s2i-play` a link to the directory?

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
+= Container Quickstarts by Red Hat's Community of Practice
+
+This repository is meant to help bootstrap users of OpenShift Container Platform get started in building and using Source-to-Image to build applications to run in OpenShift.
```
Comment (sabre1041) [inline README.adoc]: Should this sentence be the following?

"This repository is meant to help bootstrap users of **the** OpenShift Container Platform **to** get started"

**Comment by etsauer** [comment]
@sabre1041 I want to use the PR to discuss structural changes.

**Comment by etsauer** [comment]
@sabre1041 fixed.

**Comment by sabre1041** [comment]
@etsauer since the example images all use markdown format, would you be able to chance this file to .md?

**Comment by etsauer** [comment]
@sabre1041 converted.

**Comment by sabre1041** [comment]
lgtm

**Pull request merged**